README.md

```diff
@@ -1,2 +1,71 @@
-# vue-cli-plugin-vuedock
-Docker plugin for the VueCLI
+# VueDock
+> Docker plugin for the Vue CLI 3
+
+[![Build Status](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock.svg?branch=master)](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock)
+
+## Instructions
+
+#### Development
+
+Dockerized vue-cli
+
+``` bash
+# Build image
+docker build -t vuedock --target vuecli .
+
+# Print help
+docker run --rm vuedock
+
+# Create sample app
+docker run --rm -it \
+  -v "$(pwd)":/vuedock \
+vuedock sh
+
+# Create sample app
+vue create -d app \
+  && cd app \
+  && npm install --save-dev file:/vuedock \
+  && vue invoke vue-cli-plugin-vuedock
+```
+
+#### CI
+
+``` bash
+# Build image
+docker build -t vuedock .
+
+# Dgoss Tests (quick smoke test to check if docker files were generated)
```

Comment:
Could you add a one liner on what dgoss is, a link to its documentation and what we use it for?

Reply:
Done.